CHANGELOG.rst

```diff
@@ -117,6 +117,8 @@ in development
   (``required: false``). (improvement)
 
   Reported by nmlaudy. #3468 #3469
+* Removing empty pack config.yaml so warning isn't thrown by default now that deprecation warning
+  is in place. (fix)
```

Comment:
It's a fix :trollface:
Files missing from the previous PR which introduced that deprecation notice :)